src/py/flwr/server/app.py

```diff
@@ -280,25 +304,40 @@ def run_server() -> None:
         state_factory=state_factory,
     )
 
+    grpc_servers = [driver_server]
+    bckg_threads = []
+
     # Start Fleet API
-    fleet_server: grpc.Server = _run_fleet_api_grpc_bidi(
-        address=args.fleet_api_address,
-        state_factory=state_factory,
-    )
+    if args.fleet_api_type == "rest":
+        fleet_thread = threading.Thread(
+            target=_run_fleet_api_rest,
+            args=(args.rest_fleet_api_address, state_factory),
+        )
+        fleet_thread.start()
+        bckg_threads.append(fleet_thread)
+    elif args.fleet_api_type == "grpc":
+        fleet_server = _run_fleet_api_grpc_bidi(
+            address=args.grpc_fleet_api_address,
+            state_factory=state_factory,
+        )
+        grpc_servers.append(fleet_server)
```

Comment:
Rest seems to have preceded before grpc. Should we fail if somewhere if both options are passed?

Reply:
It's not possible to pass both options because they are defined as mutually exclusive.

Reply:
👍🏼